modules/ACASLabelSequences/src/server/routes/ACASLabelSequencesRoutes.coffee

```diff
@@ -1,6 +1,5 @@
 exports.setupAPIRoutes = (app) ->
 	app.get '/api/labelSequences', exports.getAllLabelSequences
-	app.get '/api/labelSequences/reagents/codename', exports.getReagentByCodename
```

Comment:
I removed it because `exports.getReagentByCodename` function didn't exist in this file and new express throws an error when a route function doesn't exist.  The route `/api/labelSequences/reagents/codename` actually exists in "TemplateModuleRoutes.coffee" FWIW